src/dcp/dcp.c

```diff
@@ -536,6 +664,7 @@ int main(int argc, char** argv)
                     rc = daos_cont_open(src_poh, cuuid, DAOS_COO_RW, &dst_coh, &co_info, NULL);
                     if (rc != 0) {
                         MFU_LOG(MFU_LOG_ERR, "Failed to open DFS2 container");
+                        daos_rc = 1;
```

Comment:
If we hit an error here, is the bcast below still valid?

Reply:
You are right, the bcast might not be valid, and that is one such issue I ran into. My latest updates have refactored the way this works so that the error checking is propagated among the processes.